core/src/epicli/data/common/validation/configuration/logstash.yml

```diff
@@ -0,0 +1 @@
+$ref: '#/definitions/unvalidated_specification'
```

Comment:
I'd suggest adding some validation here (if you still want to keep version definitions in the doc exposed to users). :+1: